config/functions.cfg

```diff
@@ -56,6 +56,9 @@ function check_variables {
   #Check if CUSTOM_HOME exists
   if ! [ -d "$CUSTOM_HOME" ]; then echo -e "${RED}Please configure your variables first ! (config/variables.cfg --> CUSTOM_HOME & CUSTOM_USER)${NC}"; exit; fi
 
+  INST_SET=$(lscpu | grep -i Flags)
```

Comment:
kind of duplicated if because we will have this in the next node release. I would remove this because we might run observers without SSU 4.2 on the metachain.

Reply:
Removed